chia/wallet/puzzles/tails.py

```diff
@@ -108,7 +108,7 @@ async def generate_issuance_bundle(cls, wallet, _: Dict, amount: uint64) -> Tupl
                 False,
             )
 
-        return tx_record, SpendBundle.aggregate([tx_record.spend_bundle, signed_eve_spend])
+        return tx_record, SpendBundle.aggregate([tx_record.spend_bundle, signed_eve_spend]), origin_id
```

Comment:
Origin ID is not the thing we should be returning here (although it might work anyway), I think we want to return the `tail.get_tree_hash()` instead.

Reply:
> I think we want to return the `tail.get_tree_hash()` instead.

@Quexington I don't see how that would help? The other way to fix this would be to create the lineage storage inside of `generate_issuance_bundle`, see [0d4c8a0](https://github.com/Chia-Network/chia-blockchain/pull/10791/commits/0d4c8a00a692e351474dbdb9bb065841d6ed3328)